underfs/pom.xml

```diff
@@ -39,7 +39,7 @@
       <dependency>
         <groupId>net.java.dev.jets3t</groupId>
         <artifactId>jets3t</artifactId>
-        <version>0.8.1</version>
+        <version>0.9.4</version>
```

Comment:
v0.9.x jets3t has performance issues when doing seeks. See: https://bitbucket.org/jmurty/jets3t/commits/b41fbb3aa606
Is it possible to use the S3A client with V4?